projects/angular-auth-oidc-client/src/lib/utils/url/url.service.ts

```diff
@@ -439,7 +439,7 @@ export class UrlService {
         const authWellKnownEndPoints = this.storagePersistenceService.read('authWellKnownEndPoints', configuration);
 
         if (authWellKnownEndPoints) {
-          return this.createAuthorizeUrl(codeChallenge, silentRenewUrl, nonce, state, configuration, 'none', customParams);
+          return this.createAuthorizeUrl(codeChallenge, silentRenewUrl, nonce, state, configuration,configuration.silentRenewPrompt, customParams);
```

Comment:
We are passing the configuration into the method already. No need for this parameter and we can extract the param in the method, can't we?

Reply:
I want to be sure its only applied when in silent renew mode so I think its better this way.

Reply:
Thanks for your comment. I do not understand why this would be the case when reading the parameter in the method. Can you elaborate?